### PR TITLE
Fix inventories-from-projects when running in Kubernetes

### PR DESCRIPTION
**Pull request by shanemcd**
Related: https://github.com/ansible/awx/issues/9704

Will also require a new release of the operator which will contain https://github.com/ansible/awx-operator/pull/155

### CODE REVIEW

**Review by kdelee — APPROVED** [review]
Looks sane

File: awx/main/tasks.py [hunk 1]
```diff
@@ -3010,7 +3011,7 @@ def run(self):
             return self._run_internal(receptor_ctl)
         finally:
             # Make sure to always release the work unit if we established it
-            if self.unit_id is not None:
+            if self.unit_id is not None and not settings.AWX_CONTAINER_GROUP_KEEP_POD:
```
Comment (chrismeyersfsu) [inline awx/main/tasks.py:3014]: Thank you. These little debugging flags will be priceless.

File: awx/main/tests/functional/conftest.py [hunk 2]
```diff
@@ -829,5 +829,5 @@ def r(N, jt_kwargs=None, prompts=None, spawn=False):
 
 
 @pytest.fixture
-def execution_environment(organization):
-    return ExecutionEnvironment.objects.create(name="test-ee", description="test-ee", organization=organization)
+def execution_environment():
+    return ExecutionEnvironment.objects.create(name="test-ee", description="test-ee", managed_by_tower=True)
```
Comment (chrismeyersfsu) [inline awx/main/tests/functional/conftest.py:833]: https://github.com/ansible/awx/blob/9a61c4368784c421e47551e4175020ced116f56e/awx/main/management/commands/create_preload_data.py#L72 :+1:

**Review by chrismeyersfsu — APPROVED** [review]

File: awx_collection/test/awx/test_completeness.py [hunk 3]
```diff
@@ -157,7 +157,7 @@ def determine_state(module_id, endpoint, module, parameter, api_option, module_o
     return 'OK'
 
 
-def test_completeness(collection_import, request, admin_user, job_template):
+def test_completeness(collection_import, request, admin_user, job_template, execution_environment):
```
Comment (beeankha) [inline awx_collection/test/awx/test_completeness.py:160]: :+1:

**Review by beeankha — APPROVED** [review]

**Comment by MaximilianMeister** [comment]
@shanemcd thanks for fixing this, can you estimate more or less when this is included in a next release?

**Comment by shanemcd** [comment]
@MaximilianMeister It will probably be early-mid next week.

**Pull request merged**